client/services/code_preprocessors/PythonCodePreprocessorServiceSpec.js

```diff
@@ -34,6 +34,29 @@ describe('PythonCodePreprocessorService', function() {
       'PerformanceTestObjectFactory');
   }));
 
+  describe('_jsonVariableToPython', function() {
+    it('should correctly convert a json String to a Python string', function() {
+      expect(
+        PythonCodePreprocessorService._jsonVariableToPython('stringify')
+      ).toEqual("'stringify'");
+    });
+
+    it('should correctly convert a json Array to a Python array (or a string version)'
```

Comment:
Sorry, it's a string representation of a Python array. I've amended the comment to be clearer.